tests/sqllogictests/suites/base/03_common/03_0004_select_order_by.test

```diff
@@ -113,6 +113,18 @@ INSERT INTO t3 VALUES(1, [1,2,3]), (2, [1,2,4]), (3, []), (4, [3,4,5]), (5, [4])
 statement ok
 DROP TABLE t3
 
+query I
+SELECT number FROM numbers(3) ORDER BY number LIMIT 214748364900;
```

Comment:
If limit is too large, we don't need to push the limit inside the input.

I think we also need to change it in `rule_push_down_limit_xxx.rs`